Sources/XcodeGenKit/InfoPlistGenerator.swift

```diff
@@ -8,20 +8,26 @@ public class InfoPlistGenerator {
      Default info plist attributes taken from:
      /Applications/Xcode.app/Contents/Developer/Library/Xcode/Templates/Project Templates/Base/Base_DefinitionsInfoPlist.xctemplate/TemplateInfo.plist
      */
-    var defaultInfoPlist: [String: Any] = {
+    private func generateDefaultInfoPlist(target: Target) -> [String: Any] {
         var dictionary: [String: Any] = [:]
         dictionary["CFBundleIdentifier"] = "$(PRODUCT_BUNDLE_IDENTIFIER)"
         dictionary["CFBundleInfoDictionaryVersion"] = "6.0"
-        dictionary["CFBundleExecutable"] = "$(EXECUTABLE_NAME)"
+
         dictionary["CFBundleName"] = "$(PRODUCT_NAME)"
         dictionary["CFBundleDevelopmentRegion"] = "$(DEVELOPMENT_LANGUAGE)"
         dictionary["CFBundleShortVersionString"] = "1.0"
         dictionary["CFBundleVersion"] = "1"
+
+        // Bundles should not contain any CFBundleExecutable otherwise they will be rejected when uploading.
```

Comment:
Could you update ProjectSpec documentation too?
https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md#target

> The following properties are generated automatically